payment-manager/src/test/java/com/qpay/paymentmanager/repository/WalletRepositoryTest.java

```diff
@@ -0,0 +1,69 @@
+package com.qpay.paymentmanager.repository;
+
+import com.qpay.paymentmanager.model.entity.wallet.WalletEntity;
+import com.qpay.paymentmanager.model.usertype.UserType;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.boot.test.autoconfigure.orm.jpa.TestEntityManager;
+import org.springframework.test.context.ActiveProfiles;
+import java.math.BigDecimal;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DataJpaTest()
+@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
+@ActiveProfiles(value = "test")
+class WalletRepositoryTest {
+
+    @Autowired
+    private WalletRepository walletRepository;
+
+    @Autowired
+    private TestEntityManager testEntityManager;
+
+    @Test
+    void should_returnWalletsByUserIdAndUsertype() {
+        // given
+        var userId = 1L;
+        var userType = UserType.CUSTOMER;
+        var wallet1 = testEntityManager.persistAndFlush(
+                WalletEntity.builder()
+                        .balance(BigDecimal.valueOf(100))
+                        .name("wallet1")
+                        .userId(userId)
+                        .userType(userType)
+                        .build()
+        );
+        var wallet2 = testEntityManager.persistAndFlush(
+                WalletEntity.builder()
+                        .balance(BigDecimal.valueOf(500))
+                        .name("wallet2")
+                        .userId(userId)
+                        .userType(userType)
+                        .build()
+        );
+        var wallet3 = testEntityManager.persistAndFlush(
```

Comment:
```suggestion
        testEntityManager.persistAndFlush(
```

you dont have to assign it to the var, you dont use it anyway